src/chain/state.rs

```diff
@@ -163,8 +158,26 @@ impl State {
         Ok(())
     }
 
+    /// Write the initial state to the given path on disk
+    fn write_initial_state(path: &Path) -> Result<Self, Error> {
+        let mut consensus_state = consensus::State::default();
+
+        // TODO(tarcieri): correct upstream `tendermint-rs` default height to 0
```

Comment:
Would be good to change this upstream in `tendermint-rs`, then we can just rely on `consensus::State::default()`